pkg/agent/openflow/client.go

```diff
@@ -545,11 +545,14 @@ func (c *client) InstallGatewayFlows() error {
 }
 
 func (c *client) InstallDefaultTunnelFlows() error {
-	flow := c.tunnelClassifierFlow(config.DefaultTunOFPort, cookie.Default)
-	if err := c.ofEntryOperations.Add(flow); err != nil {
+	flows := []binding.Flow{
+		c.tunnelClassifierFlow(config.DefaultTunOFPort, cookie.Default),
+		c.l2ForwardCalcFlow(globalVirtualMAC, config.DefaultTunOFPort, true, cookie.Default),
```

Comment:
`globalVirtualMAC` is also used in some other cases like Windows SNAT as the mark to replace the dst MAC. Do you think we could unify the usage, e.g., using this MAC for tunnel traffic, and using `macRewriteMark` to rewrite dstMAC in any Node?

Reply:
That (using macRewriteMark to indicate MAC rewrite) makes sense to me. But I am not familiar with all other cases. Would you make the change?

Reply:
OK, I would make that change.